### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by davidkuridza** [Byron/google-apis-rs#500] [Q&A] [answered]
Our pipelines are failing due to updated yup-oauth2, the fix in `google-apis-rs` is already in main with #491. Would it be possible to create a new release?

Thank you.

**Comment by Byron** [comment]
As I try to avoid spamming crates.io with frequent releases, I prefer to not create an all-out release at this time.

However, if you share the APIs you are actually using, I can selectively publish them.

**Comment by davidkuridza** [comment]
Got it, thanks! We are using the Calendar API (`google-calendar3`).

**Reply by Byron** [reply]
It's done: https://crates.io/crates/google-calendar3/5.0.5+20240328

**Reply by davidkuridza** [reply]
Hm, there appears to be another issue:

```
the trait `google_calendar3::google_apis_common::GetToken` is not implemented for `yup_oauth2::authenticator::Authenticator<google_calendar3::hyper_rustls::HttpsConnector<google_calendar3::hyper::client::HttpConnector>>
```

I believe a new release of `google_apis_common` is needed as well?

**Reply by Byron** [reply, marked as answer]
Thanks, let's hope this works: https://github.com/Byron/google-apis-rs/releases/tag/google-apis-common-v6.0.4 .

**Reply by davidkuridza** [reply]
Works, thank you very much!
